### ISSUE TITLE
Feature request: print found libraries when running ./waf configure --verbose

### ISSUE THREAD

**Issue by JoostHuizinga** [sferes2/sferes2#45]
When running ./waf configure, sferes will print which libraries and header files are being checked. e.g.:

```
Checking boost includes                  : 1_61 
Checking boost libs                      : ok 
Checking Intel TBB includes              : ok 
Checking for Eigen                       : ok 
```

I would like to get some extra information when the --verbose option is passed. Currently, passing --verbose to ./waf configure prints this:

```
Checking boost includes                  : 1_61 
	path : /usr/local/include 
Checking boost libs                      : ok 
	path : /usr/local/lib 
	libs : ['boost_serialization-mt', 'boost_filesystem-mt', 'boost_system-mt', 'boost_unit_test_framework-mt', 'boost_program_options-mt', 'boost_graph-mt', 'boost_thread-mt', 'boost_regex-mt'] 
Checking Intel TBB includes              : ok 
Checking for Eigen                       : ok 
```

I would like to know at which paths the TBB and Eigen libraries where found.

As for the problem that raised this issue, I did not think I had TBB installed on my laptop, but apparently sferes does find an implementation. Unfortunately, the implementation I have installed fails to link.

**Comment by costashatz** [comment]
Eigen has no libraries, just header files. For TBB, I just checked and we have an issue in our implementation; i.e. we check for the headers, but not for libs -- nevertheless, we link to the libs. I will fix it. Thanks for pointing out!

**Comment by costashatz** [comment]
Fixed in #45...
